packages/components/pie-toast-provider/src/toaster.ts

```diff
@@ -0,0 +1,35 @@
+import { type PieToastProvider } from './index';
+import { type ExtendedToastProps } from './defs';
+
+/**
+ * Singleton toaster interface for global access.
+ */
+export const toaster = {
+    _getToastProvider (): PieToastProvider | null {
+        const toastProviders = document.querySelectorAll('pie-toast-provider');
+
+        if (toastProviders.length === 0) {
+            console.error('ToastProvider is not initialized.');
```

Comment:
```suggestion
            console.error('The pie-toast component requires a pie-toast-provider element present in the DOM.');
```
I believe this would be more helpful, as it informs what is missing or expected.